CMakeLists.txt

```diff
@@ -52,32 +62,29 @@ add_subdirectory(mbed-os)
 ### End of block
 ######################################################################################################
 
-### Set up your project name (necessary everytime)
-project(MbedCEHelloWorld)
-
 ### add executable (necessary everytime)
-add_executable(${CMAKE_PROJECT_NAME} main.cpp)
+# This will compile an elf file named the same as the project with the given source files.
+add_executable(${PROJECT_NAME} main.cpp)
 
 ######################################################################################################
 ### Link libraries block
 #[[For more about this configuration visit wiki page MbedOS-configuration
 https://github.com/mbed-ce/mbed-os/wiki/MbedOS-configuration#configuration-via-cmake-files]]
 
 #[[link MbedOS and its libraries (necessary everytime)]]
-target_link_libraries(${CMAKE_PROJECT_NAME}  mbed-os) 
+target_link_libraries(${PROJECT_NAME} mbed-os)
 
 ### link user library (if needed)
-#target_link_libraries(${CMAKE_PROJECT_NAME} YourLibrary) 
+#add_subdirectory(YourLibrary)
```

Comment:
One should be deleted probably
https://github.com/mbed-ce/mbed-ce-hello-world/blob/aa1090edcbfe249744bb4fab6b2b1145f16c5f8f/CMakeLists.txt#L58-L60

Reply:
oh oops you right